### ISSUE TITLE
[Feature] Add the possibility to add "horizon" as a feature

### ISSUE THREAD

**Issue by JanMaartenvanDoorn** [OpenSTEF/openstef#183]
WHAT: Add the possibility to add "horizon" as a feature. In some cases it is nice to have "Horizon" passed down to the model as a feature. Here we implement this option.

HOW:
- Use the feature_names list to specify that the Horizon column has to be passed down
- If the "horizon" fetaure is not in feature names it should be excluded
- Make unit tests to properly test each case

TEST:
- "horizon" can be added as a feature on request

**Comment by Davidswinkels** [comment]
depends on: #207
